.github/workflows/test.yaml

```diff
@@ -64,15 +64,13 @@ jobs:
       - name: Install the package
         run: |
           cd $PROJECT_ROOT
-          pip install ".[doc]"
+          pip install .
           python -c "import ${{ matrix.package-name }}"
       # Docs
       - name: Build the documentation
-        env:
-          SPHINXOPTS: -W --keep-going
         run: |
-          cd "$PROJECT_ROOT/docs"
-          make html
+          cd $PROJECT_ROOT
+          hatch run docs:build -W
```

Comment:
this will build with uv, so it won’t use any of the caches we set up (`pip`, `pre-commit`)

so to speed up our CI, we should switch this workflow to uv as well.